treetoreads.py

```diff
@@ -74,14 +79,18 @@ def test_deps(self):
                              and in your path for TreeToReads to
                              generate reads. Art not found.
                              TTR will only generate mutated genomes. \n''')
-            self.no_art = 1
+            self.no_art = True
         else:
-            self.no_art = 0
+            self.no_art = False
         if not dendropy.__version__.startswith('4'):
             sys.stderr.write('''ERROR: Please upgrade the python package dendropy to version 4,
                                 using 'pip install dendropy --upgrade'.
                                 Exiting\n''')
             self._exit_handler()
+        if call(['which', 'pigz'], stdout=open('/dev/null', 'w')) == 1:
```

Comment:
this should be 
```if call(['which', 'pigz'], stdout=open('/dev/null', 'w')) == 0: ```
otherwise it tries to use pigz when it is not installed

Reply:
Ah, bad copypasting on my part